jest.config.js

```diff
@@ -1,3 +1,5 @@
+process.env.TZ = 'UTC';
```

Comment:
This sets the timezone for our tests to UTC, I didn't want tests to fail once BST ends.